packages/react-router-architect/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@react-router/architect",
-  "version": "2.9.0-pre.0",
+  "version": "6.26.2",
```

Comment:
Align these all to 6.x so they all bump to v7 together when we do the first alpha release